apps/sertra/components/mod-selector/list.tsx

```diff
@@ -16,7 +15,14 @@ export const ModRowInfo: React.FC<ModRowInfoProps> = ({
   return (
     <>
       <div className={styles.colSmall}>
-        <Image className={styles.rowIcon} alt="mod icon" src={iconUrl} />
+        {/* eslint-disable-next-line @next/next/no-img-element */}
+        <img
+          height={"32px"}
```

Comment:
I prefer `height="32px"` for plain string values. But I guess this is something that the linter should catch on, so feel free to leave it as is. Either way you could ensure that we have a ticket for linting in this project, and that this issue is on the list of things we need a house rule for.

Reply:
Those values are actually already set in the styles.rowIcon, so I'll remove them. Those were added because linter wanted to use Image component which would've required width and height props.

Reply:
Well that solves the issue, but I'd appreciate if you'd create the linting ticket anyway. Feel free to resolve this conversation once you've read this comment.

Reply:
I know my IDEs at least auto-insert the braces regardless of value type, so as with any other formatting rules it'd be best if whatever formatting is linted for is also automated. In other words, a lint rule alone is not a sufficient change, the formatting should also be handled by prettier. If both can't be done then I don't think we should have a lint rule.